docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -478,6 +541,72 @@ The treatment algorithm is described in this :download:`word document found here
 
 **How to assign treatment effects:**
 
+Each treatment category has a hazard ratio associated with it both for progression-free survival and overall survival relative to the overall progression-free survival and overall survival of their demographic group as a whole. There are separate hazard ratios for the first line of treatment and later lines of treatment for each treatment category. Additionally, the hazard ratios for the isatuxamib- and daratumumab-containing treatment categories vary based on retreatment status (:code:`retreatment == True`). The hazard ratios are shown in the tables below.
+
+.. list-table:: First line of treatment hazard ratios
+   :header-rows: 1
+
+   * - Treatment category
+     - Progression free survival HR (95% UI)
+     - Overall survival HR (95% UI)
+   * - All categories combined
+     - 1 (reference)
+     - 1 (reference)
+   * - Isatuxamib-containing treatment category
+     - 0.429 (0.368, 0.495)
+     - 0.760 (0.645, 0.895)
+   * - Daratumumab-containing treatment category
+     - 0.429 (0.368, 0.495)
+     - 0.760 (0.645, 0.895)
+   * - Residual treatment category
+     - 1.00581 (1.0051, 1.0064)
+     - 1.0024 (1.0011, 1.0036)
+
+.. list-table:: Second and later lines of treatment hazard ratios
+   :header-rows: 1
+
+   * - Treatment category
+     - Progression free survival HR (95% UI)
+     - Overall survival HR (95% UI)
+   * - All categories combined
+     - 1 (reference)
+     - 1 (reference)
+   * - Isatuxamib-containing treatment category, :code:`retreated != True`
+     - 0.530 (0.356, 0.803)
+     - 1.116 (1.044, 1.185)
+   * - Isatuxamib-containing treatment category, :code:`retreated == True`
+     - 0.765 (0.678, 0.902)
+     - 1.232 (1.088, 1.37)
+   * - Daratumumab-containing treatment category, :code:`retreated != True`
+     - 0.217 (0.203, 0.231)
+     - 0.572 (0.551, 0.594)
+   * - Daratumumab-containing treatment category, :code:`retreated == True`
+     - 0.609 (0.601, 0.616)
+     - 0.786 (0.776, 0.797)
+   * - Residual treatment category
+     - 1.331 (1.324, 1.337)
+     - 1.181 (1.171, 1.190)
+
+A lognormal distribution of uncertainty within the uncertainty intervals reported above should be assumed. The code block below instructs how to construct a distribution for each hazard ratio so that it can be sampled from.
+
+.. code-block:: python
+
+	from numpy import log
+	from scipy.stats import norm, lognorm
+
+	q_975_stdnorm = norm().ppf(0.975)
+	mu = log(hr)
+	sigma = (log(hr_upper) - mu) / q_975_stdnorm
+	hr_distribution = lognorm(s=sigma, scale=hr)
```

Comment:
@collijk @mattkappel, FYI the sampling distribution instructions are in this PR now and should be merged soon! I believe there is an outstanding to-do to incorporate this into the treatment effects model.